From 49ee1838bffb5960e5e9bd59c0187d47203c2b44 Mon Sep 17 00:00:00 2001
From: Klaus Aehlig <aehlig@google.com>
Date: Mon, 3 Feb 2014 17:50:11 +0100
Subject: [PATCH] Fix specification of TIDiskParams

Commit 580b1fdd incorrectly assumes that disk parameters are
just the standard ones, whereas the man page explicitly states
that additional parameters can be passed as well, if they make
sense for the chosen storage type. Fix this.

Signed-off-by: Klaus Aehlig <aehlig@google.com>
Reviewed-by: Petr Pudlak <pudlak@google.com>
---
 lib/ht.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/ht.py b/lib/ht.py
index d6171ac..ee82148 100644
--- a/lib/ht.py
+++ b/lib/ht.py
@@ -570,7 +570,7 @@ TINicParams = \
                                       TMaybe(TString)))
 
 TIDiskParams = \
-    Comment("Disk parameters")(TDictOf(TElemOf(constants.IDISK_PARAMS),
+    Comment("Disk parameters")(TDictOf(TNonEmptyString,
                                        TOr(TNonEmptyString, TInt)))
 
 THypervisor = TElemOf(constants.HYPER_TYPES)
-- 
1.9.0

